src/data/mainnet/arbitrum-one-tokens-info.json

```diff
@@ -44,6 +44,7 @@
     "name": "Aave Arbitrum USDCn",
     "symbol": "aArbUSDCn",
     "decimals": 6,
+    "pegTo": "0xaf88d065e77c8cc2239327c5edb3a432268e5831",
```

Comment:
Should this token also to be in this file? Otherwise I am not sure we'd know which network to look for it in.

Reply:
I missed it a first glance, but see it now: [here](https://github.com/valora-inc/address-metadata/blob/main/src/data/mainnet/arbitrum-one-tokens-info.json#L15).